recaf-plugin/src/main/java/dev/xdark/recaf/plugin/repository/CommonPluginRepository.java

```diff
@@ -0,0 +1,25 @@
+package dev.xdark.recaf.plugin.repository;
```

Comment:
I moved all of the plugin logic that I could into the `recaf-plugin` module, with the intention to create minimal _"wrappers"_ to add on removed functionality.

This way the plugin stuff can sit in the plugin module, and the UI add-ons are just that. Minimal add-ons.